test/suites/rbac.sh

```diff
@@ -0,0 +1,125 @@
+  ! lxc project set rbac:default user.foo=bar || false # Does not have "manage-projects"
+  ! lxc project create rbac:p2 || false # Does not have "admin"
+  ! lxc delete rbac:c1 || false # Does not have "manage-containers"
+  ! LXC_LOCAL='' lxc_remote exec rbac:c1 -- true || false # Does not have "operate-containers"
+  lxc image delete "rbac:${testimage_fingerprint}" # Has "manage-images"
+  ! lxc network delete rbac:n1 || false # Does not have "manage-networks"
+  ! lxc profile delete rbac:empty || false # Does not have "manage-profiles"
+  ! lxc storage volume delete rbac:s1 v1 || false # Does not have "manage-storage-volumes"
+
+  # Re-import the test image as we just deleted it.
+  ensure_import_testimage
+
+  curl "${rbac_url}/set-perms?user=user1" -X POST -d '{"default": ["manage-networks", "view"]}'
+  ! lxc project set rbac:default user.foo=bar || false # Does not have "manage-projects"
+  ! lxc project create rbac:p2 || false # Does not have "admin"
+  ! lxc delete rbac:c1 || false # Does not have "manage-containers"
+  ! LXC_LOCAL='' lxc_remote exec rbac:c1 -- true || false # Does not have "operate-containers"
+  ! lxc image delete "rbac:${testimage_fingerprint}" || false # Does not have "manage-images"
+  lxc network delete rbac:n1 # Has "manage-networks"
+  ! lxc profile delete rbac:empty || false # Does not have "manage-profiles"
+  ! lxc storage volume delete rbac:s1 v1 || false # Does not have "manage-storage-volumes"
+
+  curl "${rbac_url}/set-perms?user=user1" -X POST -d '{"default": ["manage-profiles", "view"]}'
+  ! lxc project set rbac:default user.foo=bar || false # Does not have "manage-projects"
+  ! lxc project create rbac:p2 || false # Does not have "admin"
+  ! lxc delete rbac:c1 || false # Does not have "manage-containers"
+  ! LXC_LOCAL='' lxc_remote exec rbac:c1 -- true || false # Does not have "operate-containers"
+  ! lxc image delete "rbac:${testimage_fingerprint}" || false # Does not have "manage-images"
+  ! lxc network delete rbac:lxdbr0 || false # Does not have "manage-networks"
+  lxc profile delete rbac:empty # Has "manage-profiles"
+  ! lxc storage volume delete rbac:s1 v1 || false # Does not have "manage-storage-volumes"
+
+  curl "${rbac_url}/set-perms?user=user1" -X POST -d '{"default": ["manage-storage-volumes", "view"]}'
+  ! lxc project set rbac:default user.foo=bar || false # Does not have "manage-projects"
+  ! lxc project create rbac:p2 || false # Does not have "admin"
+  ! lxc delete rbac:c1 || false # Does not have "manage-containers"
+  ! LXC_LOCAL='' lxc_remote exec rbac:c1 -- true || false # Does not have "operate-containers"
+  ! lxc image delete "rbac:${testimage_fingerprint}" || false # Does not have "manage-images"
+  ! lxc network delete rbac:n1 || false # Does not have "manage-networks"
+  ! lxc profile delete rbac:empty || false # Does not have "manage-profiles"
+  lxc storage volume delete rbac:s1 v1 # Has "manage-storage-volumes"
+
+  lxc storage delete s1
+  lxc rm c1 -f
```

Comment:
I know some of those command aliases slipped by in the test suite but I think it's best to stick to the official command name so `/rm/delete/` here, please.

Reply:
Will do!